src/Evolution/DgSubcell/Tags/Coordinates.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Evolution/DgSubcell/Tags/Mesh.hpp"
+#include "Utilities/GetOutput.hpp"
+
+/// \cond
+class DataVector;
+template <size_t Dim>
+class Mesh;
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace evolution::dg::subcell::Tags {
+/// The coordinates in a given frame.
+template <size_t Dim, typename Frame>
+struct Coordinates : db::SimpleTag {
+  static std::string name() noexcept {
+    return get_output(Frame{}) + "Coordinates";
+  }
+  using type = tnsr::I<DataVector, Dim, Frame>;
+};
+
+/// The logical coordinates on the subcell grid
+template <size_t VolumeDim>
+struct LogicalCoordinates : Coordinates<VolumeDim, Frame::Logical>,
```

Comment:
`LogicalCoordinatesCompute`?

Reply:
This is the same name that `domain/LogicalCoordinates` uses :shrug:

Reply:
Per our recent conventions that one should probably also changed then (not in this PR of course)